internal/confmapprovider/discovery/bundle/bundle.d/receivers/redis.discovery.yaml.tmpl

```diff
@@ -0,0 +1,69 @@
+{{ receiver "redis" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)redis"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)redis" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)redis"
+  config:
+    default:
+      endpoint: {{ defaultValue }}
```

Comment:
This will be set from the discovered endpoint so I'm not sure we should be setting a default.